content/Services/RHOBS/design.md

```diff
@@ -0,0 +1,196 @@
+# Design
+
+## Introduction
+
+This document presents the general design of [the RHOBS service](README.md). Generally, the RHOBS is a managed service running [Observatorium](http://observatorium.io/) project for the Red Hat needs. The high-level overview of the single cluster Observatorium can be seen below:
+
+![img.png](../../assets/observatorium.png)
+
+Observatorium is a set of Jsonnet configuration pieces that deploy popular open-source observability backends for each signal (metric, logs and tracing) as needed. It also provides a separate API service that handles Saas-specific duties like tenancy, rate-limiting, authorization and authentication, etc.
+
+It supports [Prometheus](https://prometheus.io/) (metric) APIs thanks to [Thanos](https://thanos.io/) open source project. Logging is provided with [Loki](https://github.com/grafana/loki) APIs and backend. Tracing with (not implemented yet) with [OpenTelemetry OTLP](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/otlp.md) API for writes, [Jaeger](https://www.jaegertracing.io/) API for reads and [Tempo](https://github.com/grafana/tempo) as a backend.
+
+Those sub-projects were chosen because of their similar architecture, scalability patterns and open source practices. The key fact is that they all use object storage as the primary data store, which allows cheap long-term retention of the (mostly) immutable observability data.
+
+The RHOBS service uses Observatorium. We deploy Observatorium on multiple clusters in multiple regions. It also adds integration services and connections to Red Hat-specific dependencies. The rest of the document describes all the pieces in detail.
+
+> NOTE: This document will focus on the metric signal. Logging is only partially deployed in certain places but is not yet a critical dependency on anything. It is also deployed in logical and network separation to the metric backend, so it should not affect the metric system; thus, all signals can be described separately. Tracing is in heavy progress as well.
+
+## Metrics
+
+### Architecture
+
+The current high-level architecture of the production service looks as follows:
+
+![](../../assets/rhobs-design1.png)
+
+Legend:
+
+* Blue background: Managed by RHOBS.
+* Grey background: Third-party managed software.
```

Comment:
The S3 buckets have green backgrounds, so maybe that should be specified too (as they are run by AWS + appSRE).